.github/workflows/ci.yml

```diff
@@ -145,3 +145,17 @@ jobs:
         run: |
           git-secrets --register-aws
           git-secrets --scan
+  custom-standard-c-header:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone This Repo
+        uses: actions/checkout@v2
+      - name: Build
+        run: |
+          mkdir -p override-include
+          cp source/include/stdint.readme override-include/stdint.h
+          cmake -S test -B build/ \
+          -G "Unix Makefiles" \
+          -DBUILD_CLONE_SUBMODULES=ON \
+          -DCMAKE_C_FLAGS='-Wall -Wextra -Werror -I../override-include'
```

Comment:
Unsure if we should have -Werror since we're substituting a system header with a cut down version for this test. It might make sense for a compiler to warn about this

Reply:
With the presence of `-Werror`, we will become aware of whether our way of testing is not sufficient to mimic a C90 toolchain. In the future, if the library were to require a dependency on `stdlib.h`, the presence of `-Werror` can help us identify the conflicting typedefs (which would require us to test the C90 compliance in a different way, for example, with a toolchain that supports only C90). 
Thus, I think there is value in using the `-Werror` flag to identify issues with testing in the future.